### PR TITLE
DFPL-2167-Respondent-Details (added respondent details to smoke test)

### PR DESCRIPTION
**Pull request by abigailsmith97**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-2167

### Change description ###
added 'respondent details' to smoke test 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: playwright-e2e/tests/smoke-test.spec.ts [hunk 1]
```diff
@@ -69,6 +71,12 @@ test("Smoke Test @smoke-test", async ({
   await addApplicationDocuments.uploadDocumentSmokeTest();
   await startApplication.addApplicationDocumentsInProgress();
 
+  await page.pause();
```
Comment (Iphiclus) [inline playwright-e2e/tests/smoke-test.spec.ts:74]: await page.pause(); perhaps not needed?

**Review by Iphiclus — CHANGES_REQUESTED** [review]

File: playwright-e2e/tests/smoke-test.spec.ts [hunk 2]
```diff
@@ -69,6 +71,12 @@ test("Smoke Test @smoke-test", async ({
...
+  // Add respondents' details
+  await startApplication.respondentsDetailsLink.isVisible();
+  await startApplication.respondentDetails();
+  await respondentDetails.respondentDetailsNeeded();
```
Comment (GauravChawlaMOJ) [inline playwright-e2e/tests/smoke-test.spec.ts:78]: May be 'await startApplication.respondentsDetailsLink.isVisible();' can be added into startApplication.respondentDetails() method?

File: playwright-e2e/tests/smoke-test.spec.ts (hunk 1 above)
Comment (abigailsmith97) [reply playwright-e2e/tests/smoke-test.spec.ts:74]: ah yes! good spot, probably need to start using the breakpoints instead..

File: playwright-e2e/tests/smoke-test.spec.ts (hunk 2 above)
Comment (abigailsmith97) [reply playwright-e2e/tests/smoke-test.spec.ts:77]: i think this was actually duplicated perhaps so have removed.

**Review by Iphiclus — CHANGES_REQUESTED** [review]
Hi, great stuff, there is a missing radio button towards the end of the test.
"Do they have legal representation?" which looks to missing, which leaves the status of this section as 'in progress'. Which would stop the case being submitted as far as I am aware. Have a quick look if this needs adding?

**Review by GauravChawlaMOJ — APPROVED** [review]

**Review by Iphiclus — APPROVED** [review]

**Pull request merged**